src/common/constants.js

```diff
@@ -1,3 +1,18 @@
+// Axios constant
+export const BASE_URL = 'https://api.nasa.gov/'
+export const GALLERY_URL = 'https://images-api.nasa.gov'
+export const API_KEY = 'api_key=aCNwLvVe8gykdUxPUggWkOt3lB3RzzHoVHTy4ZQZ'
+
+export const APP_URL = {
+  BASE: 'https://api.nasa.gov/',
+  GALLERY: 'https://images-api.nasa.gov',
+}
+
+// Path
+export const MAIN_PATH = '/'
+export const GALLERY_PATH = '/gallery/'
```

Comment:
we also agreed today to combine 2 constants into one if the prefix is a repetitive one